docs/multicluster/architecture.md

```diff
@@ -0,0 +1,87 @@
+# Antrea Multi-cluster Architecture
+
+Antrea Multi-cluster controller focuses on service discovery of containers across
+multiple clusters. Its implementation leverages Kubernetes and Kubernetes native
+solutions as much as possible.
+
+Antrea Multi-cluster controller depends on Antrea as its Kubernetes CNI to do Service
```

Comment:
Is it true? Where is the dependency?

Reply:
It should be OK to be other CNI for this phase, but I suppose it's better to keep this info as we actually only verified it with Antrea, What's your suggestion?